src/modules/endpoint/sagas/loadLanguages.js

```diff
@@ -2,16 +2,34 @@
 
 import type { Saga } from 'redux-saga'
 import { call } from 'redux-saga/effects'
-import { createLanguagesEndpoint, LanguageModel } from '@integreat-app/integreat-api-client'
+import {
+  createLanguagesEndpoint,
+  LanguageModel,
+  createTunewsLanguagesEndpoint
+} from '@integreat-app/integreat-api-client'
 import type { DataContainer } from '../DataContainer'
 import determineApiUrl from '../determineApiUrl'
+import { tunewsApiUrl } from '../constants'
 
 export default function * loadLanguages (
   city: string,
   dataContainer: DataContainer,
-  forceRefresh: boolean
+  forceRefresh: boolean,
+  isTunewsContext?: boolean
 ): Saga<Array<LanguageModel>> {
-  const languagesAvailable = yield call(() => dataContainer.languagesAvailable(city))
+  if (isTunewsContext) {
```

Comment:
I'd prefere to have this in a separate saga. Tünews should not interfere with loading the main languages from the CMS.

Multiple very different APIs are called in this saga :fire:

Reply:
Good point, I made it this way because I wanted to have fetch languages in one place. But, now it's separated.